src/adr/0064-account-delegation.md

```diff
@@ -0,0 +1,48 @@
+# 64. Account Delegation
+
+Date: 2022-11-15
+
+## Status
+
+Accepted (lazy consensus).
+
+Not yet implemented.
+
+## Context
+
+The account delegation is a feature that gives another permission to access your account and vice versa.
+The user has been delegated can begin accessing that account from your own account,
+then read, send, respond to, and delete email messages... on their behalf.
+This feature will be helpful for teamwork, family, collaboration, or company department... 
```

Comment:
This feature is useful for VIP (who have secretaries), admins, etc...

Reply:
Please state that James currently supports account delegation in IMAP / SMTP through SASL OIDC as well as SASL AUTH PLAIN cf https://github.com/apache/james-project/blob/master/src/adr/0061-delegation.md